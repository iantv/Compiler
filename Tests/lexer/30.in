int a;	//it's comment
/* it's comment */