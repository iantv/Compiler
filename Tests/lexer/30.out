1	1	int		int
1	5	a		identifier
1	6	;		;
